task-01/task_01.md

```diff
@@ -32,6 +32,17 @@ rm 'WordCount\$IntSumReducer.class' 'WordCount\$TokenizerMapper.class' WordCount
 </property>
 ```
 
+1b. If you later receive an `Error: Java Heap Space`, then insert the following lines into `/opt/hadoop/etc/hadoop/mapred-site.xml`:
+
+```
+<property>
+    <name>mapreduce.map.java.opts</name>
+    <value>-Xmx4096m</value>
+</property>
+```
+This solution is outlined in the (Stack Overflow Post by User Amar)[https://stackoverflow.com/questions/15609909/error-java-heap-space]
```

Comment:
You can directly reference the user's post via https://stackoverflow.com/a/15611829.